src/vdom.ts

```diff
@@ -744,12 +744,13 @@ function createDom(
 			} else {
 				domNode = dnode.domNode;
 			}
+			// add attributes and children so we have a fully formed node prior to attach
```

Comment:
Not sure we need to add a comment for this

Reply:
It's probably worth a comment. Any place where order isn't apparent but required is worth mentioning.

Reply:
Hmmmm. Seemed pretty obvious being that it is actually directly before the code that does the attachment, but okay.

Reply:
I don’t think this currently offers much value, if you feel the comment is needed can you expand upon it over “fully formed node” which doesn’t really mean anything.

Reply:
To expand what I meant by not meaning anything, I mean as in describing why the ordering is important.

Reply:
There's a test that covers this now. If a future developer changes this they'll fail the test and can back-track through issues.